code/datums/components/plumbing/reaction_chamber.dm

```diff
@@ -23,7 +23,7 @@
 		for(var/datum/reagent/containg_reagent as anything in reagents.reagent_list)
 			if(required_reagent == containg_reagent.type)
 				has_reagent = TRUE
-				if(containg_reagent.volume < chamber.required_reagents[required_reagent])
+				if(containg_reagent.volume + CHEMICAL_QUANTISATION_LEVEL < chamber.required_reagents[required_reagent])
```

Comment:
Can you update [this ](https://github.com/tgstation/tgstation/blob/4e2a36fe7e56fad93495adf01c697f4b5de04e90/code/modules/plumbing/plumbers/plumbing_buffer.dm#L34)and [this ](https://github.com/tgstation/tgstation/blob/4e2a36fe7e56fad93495adf01c697f4b5de04e90/code/modules/plumbing/plumbers/plumbing_buffer.dm#L38)one aswell? They also depend on total volume

Reply:
Sure. I take it they're both meant to just add the `CHEMICAL_QUANTISATION_LEVEL` to it?

Reply:
Thought about it and yeah they do as they are isolated checks.